docs/STRONG_MOTION_DEPLOYMENT_AZIMUTH.md

```diff
@@ -0,0 +1,31 @@
+# Strong Motion Deployment Best Practices (azimuth/orientation and miniSEED channel names)
+
+This document acts to serve as a guide for field best practices when deploying and orienting strong motion sensors and recorders in the GeoNet sensor network. For the purpose of this document both analogue strong motion FBA sensors and FBA/MEMS recorders will be referred to as sensors. All GeoNet strong motion sites currently operate with a 200Hz sampling rate therefore their miniSEED channel naming convention will always have a prefix of HN*.  
+
+## Strong motion site types 
+
+Regarding instrument installation, there are three types of strong motion site (excluding structural arrays):
+
+1. Sites in buildings
+2. Sites in GeoNet field infrastructure (cabinet, vault, VSAT hut)
+3. Sites in boreholes
+
+### 1) Sites in buildings
+
+Sensors at sites in buildings are installed inside cases which are typically aligned with the walls of the building. The orientation of these sensors should be in line with the edges of their host cases, which are aligned with the walls of the host building. In short, these sensors should be aligned with the walls of the host building. Their orientation can then be taken from the orientation of these walls. The miniSEED channel naming for these sites will be HNZ|1|2 (axial). If the building structure happens to be aligned to true north, there will be no change to this naming schema and the azimuth will be recorded as 0 (to maintain metadata consistency/continuity).
+
+### 2) Sites in GeoNet field infrastructure (cabinet, vault, VSAT hut)
+
+Sensors at sites in GeoNet infrastructure are installed in light structures without cases. There is varying amounts of room for orienting sensors in these installations. Here it is desirable that sensors are true north-aligned, but if this is not possible or cannot be done accurately due to space constraints, an alignment with the host structure’s walls is acceptable. The orientation can then be taken from the orientation of these walls. For the majority that are true north-aligned the miniSEED channel naming will be HNZ|N|E (non-axial) and those in the rare instances that are aligned to the structure will be named HNZ|1|2 (axial).
```

Comment:
apologies, that's me being ignorant: how can a user know the orientation of GeoNet field infrastructure walls? is that indicated somewhere in the metadata? Might be useful to specify how to get the "host structure's walls" perhaps?

Reply:
The person of the field gets the true north orientation and check the horizontal con poenents within the building ?
Then informed into Delta azimuth informatrion ? 

The user doesn t need in general the buidling orientation  ~  it is assumed to be free field (no builiding interaction assumption) .

Reply:
The GeoNet infrastructure orientation is analogous to the strong motion sensor/recorder orientation (if true North alignment is not possible). We record that orientation/azimuth information in install/sensors.csv and install/recorders.csv. We don't need to tell the end user any specifics about the infrastructure orientation (as that can be inferred from the delta azimuth entries).

cc @elidana @salichon